cmd/address.go

```diff
@@ -39,12 +39,15 @@ var (
 )
 
 func doShowAddress(cmd *cobra.Command, args []string) {
-	var walletID wallet.ID
-	if err := walletID.UnmarshalHex(viper.GetString(cfgWalletID)); err != nil {
-		logger.Error("failed to parse wallet ID",
-			"err", err,
-		)
-		os.Exit(1)
+	var walletID *wallet.ID
+	hexWalletID := viper.GetString(cfgWalletID)
+	if hexWalletID != "" {
+		if err := walletID.UnmarshalHex(hexWalletID); err != nil {
```

Comment:
Won't this cause a nil dereference when the wallet ID is passed (as `walletID` is `nil` here)?

Reply:
Yes, it will.

I've fixed it.